pkg/agent/externalnode/external_node_controller.go

```diff
@@ -707,6 +707,17 @@ func (c *ExternalNodeController) removeOVSPortsAndFlows(interfaceConfig *interfa
 		klog.ErrorS(err, "Failed to move back configuration to the host interface", "hostInterface", hostIfName)
 		return err
 	}
+
+	// TODO: Handle for multiple interface
+	ok, err := util.RemoveVMSwitch()
```

Comment:
Add these cleanup logic in a separate function rather than `removeOVSPortsAndFlows`, as the currenct function is working at per interface level. The cleanup logic should be called by function `deleteExternalNode`.

Reply:
`VMSwitch` is a concept existing only on Windows, I would prefer to add the call for `RemoveVMSwitch` in external_node_controller_windows.go

Reply:
Okay. I get it. We need to have some common function like cleanUpExternalNode or cleanUpVmConfig which should perform clean up related tasks.